src/Selector.js

```diff
@@ -28,6 +30,14 @@ export default class Selector {
       this.intersection = null;
       this.selected = null;
     }
+
+    if (this.selectEvent && this.timer < 5) {
+      this.timer += 1;
+    }
+    else if (this.selectEvent && this.timer >= 5) {
+      this.timer = 0;
+      this.selectEvent = false;
+    }
```

Comment:
The `selectEvent` has to be true for the next frame for it to make effect, so I used the timer convention I used elsewhere here as well, it probably makes more sense having two boolean values instead.